source/components/recipe-details/recipe-details.js

```diff
@@ -192,7 +192,14 @@ class RecipeDetails extends YummyRecipesComponent {
       "<ul>" + ingredients + "</ul>";
 
     // Direction box
-    this.shadowRoot.querySelector(".direction-list").innerHTML = recipe.steps;
+    let directions = "";
+
+    for (let i = 0; i < recipe.steps.length; i++) {
+      directions = directions + "<li>" + recipe.steps[i] + "</li>";
```

Comment:
You can use `+=`. See below:
```suggestion
      directions += "<li>" + recipe.steps[i] + "</li>";
```

Reply:
Fixed